tests/platform/cloud/AcceptanceTests_onboardedSolution.js

```diff
@@ -43,12 +43,11 @@ gpii.tests.cloud.untrustedSettings.testDefs = [
     {
         name: "Authorize magnifier (share magnifier settings)",
         userToken: "os_gnome",
-        authDecisions: [
+        authorizations: [
```

Comment:
I think I prefer the previous name of this test, `AcceptanceTests_untrustedSettings.js`, rather than `AcceptanceTests_onboardedSolution.js`. I totally get the reason to change to match the other client-type acceptance tests. But this test really is testing the `untrusted-settings` endpoint and I think it's better not to lose that by renaming.